tests/ProtoTests/Basic/Tests/BasicClientTest.php

```diff
@@ -56,4 +57,35 @@ public function aMethodTest()
         $this->assertSame('/testing.basic.Basic/AMethod', $actualFuncCall);
         $this->assertTrue($transport->isExhausted());
     }
+
+    /** @test */
+    public function aMethodExceptionTest()
+    {
+        $transport = $this->createTransport();
+        $client = $this->createClient([
+            'transport' => $transport,
+        ]);
+        $this->assertTrue($transport->isExhausted());
+        $status = new stdClass();
+        $status->code = Code::DATA_LOSS;
+        $status->details = 'internal error';
+        $expectedExceptionMessage  = json_encode([
+            'message' => 'internal error',
+            'code' => Code::DATA_LOSS,
+            'status' => 'DATA_LOSS',
+            'details' => [],
+        ], JSON_PRETTY_PRINT);
+        $transport->addResponse(null, $status);
+        // Mock request
+        try {
+            $client->aMethod();
+            $this->fail('Expected an ApiException, but no exception was thrown.');
+        } catch (ApiException $ex) {
+            $this->assertEquals($status->code, $ex->getCode());
+            $this->assertEquals($expectedExceptionMessage , $ex->getMessage());
+        }
+// Call popReceivedCalls to ensure the stub is exhausted
```

Comment:
How hard is it to fix this comment indentation?

Reply:
Moderately hard :( As the code-formatter I'm currently using causes this.
I've got a TODO in the generation code to fix this.